extractor/src/main/java/org/schabi/newpipe/extractor/services/bandcamp/extractors/BandcampStreamExtractor.java

```diff
@@ -208,13 +210,10 @@ public String getHlsUrl() {
 
     @Override
     public List<AudioStream> getAudioStreams() {
-        final List<AudioStream> audioStreams = new ArrayList<>();
-
-        audioStreams.add(new AudioStream(
-                albumJson.getArray("trackinfo").getObject(0)
-                        .getObject("file").getString("mp3-128"),
-                MediaFormat.MP3, 128
-        ));
+        if (isNullOrEmpty(audioStreams)) {
```

Comment:
It can't be null, but even if it could, the code inside the if uses the object, so it does not make sense to check for nullity
```suggestion
        if (audioStreams.isEmpty()) {
```

Reply:
Done, I also removed this unecessary check in other stream extractors and for video streams.